src/wslfetch.sh

```diff
@@ -256,25 +256,25 @@ case "$distro" in
 			"${red}    \`/syyyyyyyyyyyyyyyyyyyyyyys+.    ${reset}"
 			"${red}      \`.:/+ooooooooooooooo+/:.\`      ${reset}");;
 	'oldfedora'|'fedora')
-		t="${light_blue}${bold}"
+		purple=$(echo -e '\e[38;5;140m')
```

Comment:
Also this should go in header

Reply:
Yes but it modifies the original purple that you have. Maybe I can change the name?